docs/reference/commandline/cli.md

```diff
@@ -38,6 +38,7 @@ the [installation](../../installation/index.md) instructions for your operating
 For easy reference, the following list of environment variables are supported
 by the `docker` command line:
 
+* `DOCKER_API_VERSION` The API version to use (e.g. `1.19`)
```

Comment:
Oh, sorry, didn't mean a warning on the command line (that was discussed), but something in the docs. Do you think that's redundant?